src/hugr/rewrite/outline_cfg.rs

```diff
@@ -389,15 +402,22 @@ mod test {
         for &n in blocks_to_move.iter().chain(other_blocks.iter()) {
             assert_eq!(depth(&h, n), 1);
         }
-        h.apply_rewrite(OutlineCfg::new(blocks_to_move.iter().copied()))
+        let (new_block, new_cfg) = h
+            .apply_rewrite(OutlineCfg::new(blocks_to_move.iter().copied()))
             .unwrap();
         h.validate(&PRELUDE_REGISTRY).unwrap();
-        let new_entry = h.children(h.root()).next().unwrap();
+        assert_eq!(new_block, h.children(h.root()).next().unwrap());
```

Comment:
`new_entry`, although correct, could also suggest the entry node inside the new sub-cfg, hence "renaming" to `new_block`